pkg/operator/app/rundeps.go

```diff
@@ -35,6 +35,7 @@ type RunDeps struct {
 	Workflow     *obj.Workflow
 	WorkflowDeps *WorkflowDeps
 
+	Environment       string
```

Comment:
For parity, would you think that calling this `DeploymentEnvironment` might be clearer?